extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -147,6 +148,12 @@ class AmpYoutube extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    if (getMode().runtime === 'inabox') {
+      this.user().error(
+        TAG,
+        'amp-youtube is deprecated in AMPHTML ads. See https://github.com/ampproject/amphtml/issues/21340'
+      );
+    }
```

Comment:
Do you also need a warning for non-Inabox?

Reply:
Good point.Done.